packages/bitcore-client/src/storage/mongo.ts

```diff
@@ -52,6 +52,16 @@ export class Mongo {
     await this.client.close();
   }
 
+  async testConnection() {
+    try {
+      this.client = new MongoClient(this.path, { reconnectTries: 3 });
```

Comment:
`this.client` gets overwritten in the `init` function in this file, which leaves a zombie connection to the db. This results in a command forever hanging in the terminal (e.g. `bin/wallet-list`).

Furthermore, I'm getting a new error about using unified topology. Make sure you're passing the same client opts to MongoClient as the init